java/Anton-Kozyk/src/com/zoolatech/lecture3/tasks/_2/MainValidator.java

```diff
@@ -0,0 +1,31 @@
+package com.zoolatech.lecture3.tasks._2;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class MainValidator implements Validating {
+    private List<Validator> validators = new ArrayList<>();
+    private boolean validationResult = true;
```

Comment:
I don't think that `validationResult` should be considered as a part of the MainValidator state. I guess it would be better to define it as a local variable.